kubernetes/core-pod.yml

```diff
@@ -1,30 +1,30 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: busybox-pod
+  name: core-pod
   labels:
-    app: busybox
+    app: core-app
 spec:
   containers:
-  - name: busybox-container
-    image: busybox
+  - name: core-app-container
+    image: jmatejka/happy-bank-core:1.0.0
     args:
     - /bin/sh
     - -c
-    - touch tmp/healthy; sleep infinity;
+    - touch healthy; ls; sleep infinity;
```

Comment:
As I've already mentioned, this overrides 'command' property in Dockerfile, so please remove it.